app/src/main/java/org/mozilla/fenix/addons/AddonDetailsView.kt

```diff
@@ -41,6 +41,7 @@ interface AddonDetailsInteractor {
  */
 class AddonDetailsView(
     override val containerView: View,
+    private val binding: FragmentAddOnDetailsBinding,
     private val interactor: AddonDetailsInteractor
 ) : LayoutContainer {
```

Comment:
We need to also remove all `LayoutContainer`s declarations.
After this change we probably won't need the `containerView` anymore but rather the Android context needed for resources.